apps/src/blockly/customBlocks/googleBlockly/spritelabBlocks.ts

```diff
@@ -24,11 +32,14 @@ export const blocks = {
   // Creates and returns a toggle button field. This field should be
   // added to the block after other inputs have been created.
   // miniToolboxBlocks is a backwards-compatible parameter used in CDO Blockly.
-  initializeMiniToolbox(miniToolboxBlocks, renderToolboxBeforeStack = false) {
+  initializeMiniToolbox(
+    _miniToolboxBlocks: string[],
+    renderToolboxBeforeStack = false
+  ) {
     // Function to create the flyout
-    const createFlyoutField = function (block) {
+    const createFlyoutField = function (block: Block) {
       const flyoutKey = CdoFieldFlyout.getFlyoutId(block);
-      const flyoutField = new Blockly.FieldFlyout(_, {
```

Comment:
This should be a string, for some reason (my guess is error on our end) we were passing in lodash. I set it to an empty string since this clearly isn't used for anything.